content/blog/en/version-7-dojo-widgets.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Announcing Dojo 7 Widgets
+date: 2019-08-28T08:00:00.000Z
+author: Anthony Gubler
+---
+
+Since the last release, we have been working to expand the Dojo Widgets library, while also improving the developer experience. With new advances in Dojo features, functionality, and recommended best practices, Dojo 7 was the perfect time to overhaul the Dojo Widgets library. This release makes it easier for developers to build efficient and modern web applications.
+
+For Dojo 7, we focused on improvements in several key areas:
+
+- Expanding the widget library, focusing on widgets that provide the most value to end users.
+- Refactoring all widgets as function-based and in TSX.
+- Adopt the latest recommended Dojo widget authoring patterns.
+- Improve out of the box usability of each widget.
+- Standardize widget property patterns across the library.
+- Improve the theming experience for end users.
+- Expand documentation and widget examples.
+
+## Expanded Widget Library
+
+Providing maximum value to Dojo users means focusing on providing users a strong suite of widgets out of the box. Dojo 7 includes over two dozen new widgets that developers can use to build feature-rich applications even faster. Included are specialized widget variants, data-aware widgets, and widgets for building column layouts.
+
+Dojo 7 includes a key new feature called resources which are a combination of a middleware and resource factory that enables widget authors to build data-aware widgets with a standard, easy to use, and understandable API.
+
+This new powerful feature is at the heart of the data-aware widgets that were built for Dojo 7 and we will be looking for feedback from our users to further refine the widgets in future releases.
+
+The new Card widget provides a simple and consistently styled container for content and actions.
+
+![Card Widget](assets/blog/version-7-dojo-widgets/cardWidget.png)
```

Comment:
When showcasing some of the new widgets, do we want to include images, code, both?

Also, any additional/different widgets we should show off?